src/components/PopupWithImage.js

```diff
@@ -7,9 +7,9 @@ export default class PopUpWithImage extends Popup {
 
     super(popupSelector);
 
-    /* this._name = name;
+    this._name = name;
 
-    this._link = link; */
+    this._link = link;
 
     this._popupElement = document.querySelector(popupSelector);
 
```

Comment:
Please, delete `super.setEventListeners();` from method `open` in `PopUpWithImage`


`.setEventListeners()` should be removed from `open` because it's a **public** method that should be called only **once** in `index.js` otherwise you add listeners again and again to the same elements. This can cause a **`memory leak`**